src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -96,13 +100,16 @@ class CombatLogParser extends MainCombatLogParser {
     chiJi: ChiJi,
     chiBurst: ChiBurst,
 
-
     // Legendaries / Items:
     drapeOfShame: DrapeOfShame,
     prydaz: Prydaz,
     // sephuz: Sephuz,
     velens: Velens,
     eithas: Eithas,
+    archiveoffaith: ArchiveOfFaith,
+    barbaricMindslaver: BarbaricMindslaver,
+    seaStar: SeaStar,
+    dgd: DGD,
```

Comment:
Abbreviating this makes it quite unrecognizable. Right now it's no big deal but in a few weeks/months if you look at this it may take more effort to remember what it meant, slowing down development. Please expand